Documentation/Books/Manual/Programs/Arangodump/Examples.md

```diff
@@ -15,21 +15,31 @@ directory, you can use the parameter *--overwrite true* to confirm this:
     arangodump --output-directory "dump" --overwrite true
 
 _arangodump_ will by default connect to the *_system* database using the default
-endpoint. If you want to connect to a different database or a different endpoint,
-or use authentication, you can use the following command-line options:
+endpoint. To override the endpoint, or specify a different user, use one of the
+following commandline parameters:
 
-- `--server.database <string>`: name of the database to connect to
 - `--server.endpoint <string>`: endpoint to connect to
 - `--server.username <string>`: username
 - `--server.password <string>`: password to use (omit this and you'll be prompted for the
   password)
 - `--server.authentication <bool>`: whether or not to use authentication
 
+If you want to connect to a different database or dump all databases you can additionaly
+use the following command-line options (please note that the specified user must have access
+to the databases):
+
+- `--all-databases true`: must have access to all databases, and not specify a database.
+- `--server.database <string>`: name of the database to connect to
+
```

Comment:
```suggestion

Note that the specified user must have access to the databases.

```

Perhaps make this a hint box even? Also, what happens the user has access to some, but not all databases? Does it fail in the middle of the export? Or are permissions checked for all DBs before dumping starts?